### PR TITLE
Fix bugs in example repos

### PR DESCRIPTION
**Pull request by kyscott18**
Fix small bugs in example repos that occurred as a result of recent rapid changes. Also fixes linting.

### CODE REVIEW

File: examples/ethfs/.gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@ node_modules/
 
 .env.local
 .ponder/
-generated/
```
Comment (0xOlias) [inline examples/ethfs/.gitignore:6]: Would be good to convert all the examples to use the .gitignore that I pasted into the create-ponder script

File: examples/ethfs/src/FileStore.ts [hunk 2]
```diff
@@ -15,15 +18,17 @@ ponder.on("FileStore:FileCreated", async ({ event, context }) => {
...
-      contents: await context.contracts.FileStoreFrontend.read.readFile([
-        event.transaction.to as `0x{string}`,
-        filename,
-      ]),
+      contents: await context.client.readContract({
+        abi: FileStoreFrontendAbi,
+        functionName: "readFile",
+        address: "0xBc66C61BCF49Cc3fe4E321aeCEa307F61EC57C0b",
+        args: [event.transaction.to!, filename],
+      }),
```
Comment (0xOlias) [inline examples/ethfs/src/FileStore.ts:31]: I guess this works!! Seems like a fair enough pattern.

**Review by 0xOlias — APPROVED** [review]

File: examples/ethfs/src/FileStore.ts (hunk 2 above)
Comment (kyscott18) [reply examples/ethfs/src/FileStore.ts:31]: Gets tough when you want to use multichain but I wanted to get this moving.

File: examples/ethfs/.gitignore (hunk 1 above)
Comment (kyscott18) [reply examples/ethfs/.gitignore:6]: Your example .gitignore doesn't include .ponder

**Pull request merged**